systemd/systemd.go

```diff
@@ -352,7 +352,7 @@ WantedBy={{.ServiceSystemdTarget}}
 		servicesSystemdTarget,
 		origin,
 		arch.UbuntuArchitecture(),
-		"%h",
+		"/var/lib", // Remap user-level environment to system-level for services
```

Comment:
Thanks for adding a comment here, maybe add a tiny bit more like "Remap user-level environment to system-level for services. The code will append "apps/$appname/$version" to this path"